### PR TITLE
Update GitHub Action CI Visibility docs

### PR DESCRIPTION
**Pull request by cg14823**
### What does this PR do?

Updates the GitHub Actions CI Visibility setup instructions for the new opt-in system.

### Preview

https://docs-staging.datadoghq.com/carlos.gonzalez/update-gh-docs-for-opt-in/continuous_integration/setup_pipelines/github/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by drodriguezhdez — APPROVED** [review]

File: content/en/continuous_integration/setup_pipelines/github.md [hunk 1]
```diff
@@ -35,11 +38,33 @@ The [GitHub Actions][1] integration uses a private [GitHub App][2] to collect wo
 6. Give the app a name, for example, `Datadog CI Visibility`.
 7. Click **Install GitHub App** and follow the instructions on GitHub.
 
-After the GitHub App is created and installed, recently finished GitHub Actions workflows appear in CI Visibility pages.
+### Configuring Tracing for GitHub Action
+
+After the GitHub App is created and installed, you must enable CI Visibility on the accounts and/or repositories you want visibility into.
+
+1. Go to the **[Getting Started][4]** page and clik on **GitHub**.
```
Comment (soyguijarro) [inline content/en/continuous_integration/setup_pipelines/github.md:45]:
```suggestion
1. Go to the **[Getting Started][4]** page and click on **GitHub**.
```

File: content/en/continuous_integration/setup_pipelines/github.md [hunk 2]
```diff
@@ -50,5 +75,8 @@ The [Pipelines][4] and [Pipeline Executions][5] pages populate with data after t
 [1]: https://docs.github.com/actions
 [2]: https://docs.github.com/developers/apps/getting-started-with-apps/about-apps
 [3]: https://app.datadoghq.com/account/settings#integrations/github-apps
-[4]: https://app.datadoghq.com/ci/pipelines
-[5]: https://app.datadoghq.com/ci/pipeline-executions
+[4]: https://app.datadoghq.com/ci/setup/pipeline?provider=github
```
Comment (soyguijarro) [inline content/en/continuous_integration/setup_pipelines/github.md:78]: I wonder whether we should link to the onboarding or the new settings page

Comment (cg14823) [reply content/en/continuous_integration/setup_pipelines/github.md:78]: Yes, I wasn't sure about that, as it was the initial setup I went to the onboarding but I can change it to the settings page if you think it is better

File: content/en/continuous_integration/setup_pipelines/github.md [hunk 3]
```diff
@@ -35,11 +38,33 @@ The [GitHub Actions][1] integration uses a private [GitHub App][2] to collect wo
...
+After the GitHub App is created and installed, you must enable CI Visibility on the accounts and/or repositories you want visibility into.
+
+1. Go to the **[Getting Started][4]** page and click on **GitHub**.
+2. Click on **Enable Account** for the account you want to enable.
+3. Enable CI Visibility for the whole account by clicking **Enable CI Visibility**.
+4. Alternatively you can enable only individual repositories by scrolling through the repository list and clicking the **Enable CI Visibility** toggle.
+
+Pipelines will start showing immediately after enabling CI Visibility for any account or repository.
+
+### Enabling log collection
```
Comment (cswatt) [inline content/en/continuous_integration/setup_pipelines/github.md:52]:
```suggestion
### Enable log collection
```

File: content/en/continuous_integration/setup_pipelines/github.md [hunk 4]
```diff
@@ -35,11 +38,33 @@ The [GitHub Actions][1] integration uses a private [GitHub App][2] to collect wo
...
+
+The GitHub Actions CI Visibility integration also allows automatically forwarding workflow job logs to the [Logs Product][5].
+To enable logs you can follow the following:
```
Comment (cswatt) [inline content/en/continuous_integration/setup_pipelines/github.md:55]:
```suggestion
To enable logs, follow these steps:
```

**Review by cswatt — APPROVED** [review]

**Pull request merged**